scripts/issue/799/upgrade_vaults_v1_5.py

```diff
@@ -0,0 +1,175 @@
+from eth_abi import encode_abi
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from brownie import accounts, interface, web3
+from rich.console import Console
+import json
+import os
+
+C = Console()
+
+VAULTS = registry.eth.sett_vaults
+NEW_LOGIC = registry.eth.logic.TheVault
+DEV_PROXY = registry.eth.badger_wallets.devProxyAdmin
+
+KEYS = [
+    "graviAURA",
+    "bauraBal",
+    "b80BADGER_20WBTC",
+    "b40WBTC_40DIGG_20graviAURA",
+    "bBB_a_USD",
+    "b33auraBAL_33graviAURA_33WETH"
+]
+
+HODLERS = [
+    "0xD14f076044414C255D2E82cceB1CB00fb1bBA64c",
+    "0xfe51263bd0d075dc5441e89ecd1f6d63ff41e02e",
+    "0xec4fcd1aca723f8456999c5f5d7479dbe9528c11",
+    "0x3cd3dba355776ac76a0485e728ead54092085da2",
+    "0xc3b1f7ab9fabd729cdf9e90ea54ec447f9464269",
+    "0x794783dcfcac8c1944727057a3208d8f8bb91506"
+]
+
+
+def main(queue="true", simulation="false"):
+    safe = GreatApeSafe(registry.eth.badger_wallets.dev_multisig)
+    safe.init_badger()
+
+    for key, address in VAULTS.items():
+        address = web3.toChecksumAddress(address)
```

Comment:
is this needed? should already be taken care of by https://github.com/Badger-Finance/badger-multisig/blob/09d414664dae385de3890e09015dfb3afe27cda7/helpers/addresses.py#L1013

Reply:
You are right